sys/net/network_layer/sixlowpan/ip.c

```diff
@@ -690,15 +690,7 @@ ipv6_addr_t *ipv6_addr_set_by_eui64(ipv6_addr_t *out, int if_id,
 
     if (net_if_get_eui64((net_if_eui64_t *) &out->uint8[8], if_id,
                          force_generation)) {
-#ifdef MODULE_SIXLOWPAN
-
-        if (!sixlowpan_lowpan_eui64_to_short_addr((net_if_eui64_t *)&out->uint8[8])) {
-            out->uint8[8] ^= 0x02;
-        }
-
-#else
         out->uint8[8] ^= 0x02;
-#endif
```

Comment:
Still not sure if the l/u bit conversion shall be done for 16-bit short.

From https://tools.ietf.org/html/rfc6282#section-3.2.2:

>     The universal/local bit is zero to indicate local scope.